.npmignore

```diff
@@ -0,0 +1,2 @@
+src
+.babelrc
```

Comment:
remove `.gz.` files

Reply:
Could you clarify what you mean here? There are no `.gz` files in the repo since we are no longer bundling.

Reply:
dist/bundle.js.gz

Reply:
Yeah, that's what I'm saying. That file no longer exists. I'm looking at the dist directory, and it's not there, nor will it ever return since we're not bundling.